src/source/Ice/SocketConnection.c

```diff
@@ -96,54 +82,67 @@ STATUS freeSocketConnection(PSocketConnection* ppSocketConnection)
     return retStatus;
 }
 
-STATUS socketConnectionInitSecureConnection(PSocketConnection pSocketConnection, BOOL isServer)
+static STATUS tlsSessionOutBoundPacket(UINT64 customData, PBYTE pBuffer, UINT32 bufferLen)
```

Comment:
remove static keyword - not needed for function in C

Reply:
Why is that? Because my main purpose here is to avoid name conflicts. Since this is just a callback, I thought it was appropriate to keep the scope within the file only.

Reply:
First, static has no bearing on the name conflicts - those are compile time issues. The class modifier in this case is a linker directive to mangle the export name to make it local class. Second, many compilers will not support this as this is not C spec.

Reply:
Ah I see, that makes sense. I'll update it.